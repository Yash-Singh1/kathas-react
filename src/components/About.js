import React, { Component } from 'react';
import NavigationBar from './NavigationBar';

class About extends Component {
  render() {
    return (
      <NavigationBar>
        <h1>About</h1>
        <p>
          Kathas is a website based on the original{' '}
          <a href='http://kathas.github.io/'>Kathas</a> that is rewritten in
          React
        </p>
      </NavigationBar>
    );
  }
}

export default About;
