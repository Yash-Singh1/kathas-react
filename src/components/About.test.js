import React from 'react';
import ReactRouterDOM from 'react-router-dom';
import { shallow } from 'enzyme';
import About from './About';
jest.mock('react-router-dom');

describe('About', () => {
  const about = shallow(<About />);

  it('renders properly', () => {
    expect(about.html()).toMatchSnapshot();
  });

  it('has a link', () => {
    expect(ReactRouterDOM.Link.render).toHaveBeenCalled();
  });
});
