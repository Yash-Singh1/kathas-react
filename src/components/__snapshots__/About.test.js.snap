// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders properly 1`] = `
<NavigationBar>
  <h1>
    About
  </h1>
  <p>
    Kathas is a website based on the original
     
    <a
      href="http://kathas.github.io/"
    >
      Kathas
    </a>
     that is rewritten in React
  </p>
</NavigationBar>
`;
